composer.json

```diff
@@ -22,8 +22,8 @@
     },
     "require-dev": {
         "matthiasnoback/symfony-dependency-injection-test": "^3.1|^4.0",
-        "nyholm/symfony-bundle-test": "^1.6",
-        "symfony/phpunit-bridge": "^5.0",
+        "nyholm/symfony-bundle-test": "dev-master",
```

Comment:
This can be reverted. I'll release a new version now.

Reply:
cool